README.md

```diff
@@ -229,6 +229,30 @@ end
 
 Note that InSpec AWS assumes full compatibility with the underlying AWS SDK and unsupported operations will cause failures.  Therefore, depending on the external provider implementation your mileage may vary!
 
+### `aws_retry_limit` and `aws_retry_backoff`
+
+In certain cases AWS may implement rate limiting. In order to mitigate this issue the `Retry Limit` and `Retry Backoff` can be set in two ways:
+#### 1) Environment Variables
+Setting `AWS_RETRY_LIMIT` and `AWS_RETRY_BACKOFF` environment variables will be implemented at session level
+
+```bash
+export AWS_RETRY_LIMIT=5
+export AWS_RETRY_BACKOFF=5
+```
+#### 2) Inspec Control
+Certain Inspec AWS resources may support setting the Retry Limit and Retry Backoff as shown below.
+
+```
+  describe aws_config_recorder(recorder_name: aws_config_recorder_name, aws_retry_limit=5, aws_retry_backoff=5) do
+    it { should exist }
+    its('recorder_name') { should eq aws_config_recorder_name }
+  end
+```
+
+_NB. Check the AWS Inspec Resource Documentation to verify if this is supported._
```

Comment:
Has documentation been updated?

Reply:
I've removed this comment, as you mentioned above these options are supported for all controls.